spharpy/samplings/samplings.py

```diff
@@ -487,12 +490,12 @@ def equiangular(n_points=None, n_max=None, radius=1.):
 
     # repeat and normalize sampling weights
     w = np.tile(w, n_phi)
-    w = w / np.sum(w)
```

Comment:
same here, fix documentation output to samplingSphere and tell about normalization of the sphere weights